### DISCUSSION TITLE
Automations: Trigger on value field

### DISCUSSION THREAD

**Discussion by poudenes** [firefly-iii/firefly-iii#8443] [Firefly III]
Hi All,

IS there a way that you can add a Trigger with a Trigger on Value that is not case sensitive?
I have some automations and add more then 1 trigger:

Destination account name contains: Albert Heijn
Destination account name contains: ALBERT HEIJN

Some shops are different on Account. Some with capital letters some not.

**Comment by JC5** [comment]
https://docs.firefly-iii.org/references/firefly-iii/rule-triggers/

This is database dependent, for the most part.

**Comment by poudenes** [comment]
I saw that already. Sorry didn't mention it in my first post. And that makes it hard to do it with coding, because in dB it can be many things

AlBeRt HIJen, ALBERT heijn, albert HEIJN... etc etc
